### DISCUSSION TITLE
where do i put token

### DISCUSSION THREAD

**Discussion by NotYourRusher** [Cog-Creators/Red-DiscordBot#5925] [Q&A] [answered]
WHERE DE FOK DO I PUT TOKEN

**Comment by L7NEG** [comment]
Bruh put it in c/users/{ur pc name folder}/appdata/local/redbot/redbot/then ur bot folder/then u will see settings.json and then done put tge token in it

**Reply by yamikaitou** [reply]
Please do not advise people to modify files, that is an unsupported way and users can break things doing that. Plus, your path may be different than his, I know mine is

**Reply by L7NEG** [reply]
> Please do not advise people to modify files, that is an unsupported way and users can break things doing that. Plus, your path may be different than his, I know mine is

Oh ok bro im sorry

**Comment by yamikaitou** [comment, marked as answer]
You do not put it anywhere. During the install process, Red will ask you for it. You can find the install instructions on our docs at https://docs.discord.red/en/stable/

**Reply by L7NEG** [reply]
Bro i know it but if he want to change it then he do it as i typed it it is more quicker 

> You do not put it anywhere. During the install process, Red will ask you for it. You can find the install instructions on our docs at https://docs.discord.red/en/stable/

**Reply by yamikaitou** [reply]
No, the quicker way is to use `redbot --edit` to change the token

**Reply by L7NEG** [reply]
> No, the quicker way is to use `redbot --edit` to change the token

Oh ok bro u are right btw i sometimes edit the audio settings to 1000 and it works lel but not perment
